src/components/common/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactNode } from 'react';
+
+import SpaceClubHomeLogo from '../SpaceClubHomeLogo/SpaceClubHomeLogo';
+import {
+  ChildrenStyled,
+  HeaderContainerStyled,
+  LogoWrapperStyled,
+  OptionWrapperStyled,
+} from './Header.style';
+
+interface HeaderProps {
+  page: 'main' | 'club';
+  children?: ReactNode;
+}
+
+const Header = ({ page, children }: HeaderProps) => {
+  return (
+    <HeaderContainerStyled>
+      <LogoWrapperStyled>
+        <SpaceClubHomeLogo />
+      </LogoWrapperStyled>
+      <OptionWrapperStyled page={page}>
+        <ChildrenStyled page={page}>{children}</ChildrenStyled>
+      </OptionWrapperStyled>
```

Comment:
```suggestion
      <ChildrenContainerStyled>{children}</ChildrenStyled>
```
제가 위에서 리뷰한걸 적용하면 이런 구조가 되겠네요!